gemrb/core/Interface.cpp

```diff
@@ -631,8 +628,6 @@ Interface::~Interface() noexcept
 
 	// Removing all stuff from Cache, except bifs
 	if (!config.KeepCache) DelTree(config.CachePath, true);
-	
-	VideoDriver.reset();
```

Comment:
can you put this in CocoaWrapper.mm around line 210 to avoid the shutdown hang?

Reply:
There you go